### DISCUSSION TITLE
names of the features

### DISCUSSION THREAD

**Discussion by mishabounty** [pyg-team/pytorch_geometric#6665] [General]
Hi there! I use the CORA dataset to train the model and explain it using model interpretation methods (torch_geometric.explain). But in order to fully understand the operation of the model, the names of the features that are encoded in the tensor data.x (cora.x) are missing. Where can I find a dictionary with the names of the encoded features?
For example, as an explanation, I get the importance of CORA features (top 10). But I want to know the real names of the signs (1263, 1355, ...).
![image](https://user-images.githubusercontent.com/76114881/218128943-c9c590de-1b53-402d-9e51-df81e9e1725f.png)

**Comment by rusty1s** [comment]
These are sadly not available, since this dataset is only available as a pre-processed version. Here, the features represent a bag-of-words from the abstract of papers.

**Reply by mayabechlerspeicher** [reply]
Could you please explain why the fact the dataset is available as a pre-processed version and the features represent a bag-of-words from the abstract of papers makes the features not available?

**Reply by rusty1s** [reply]
Because I haven't done the pre-processing and I don't have access to the mapping from the 0/1 matrix to the raw words.

**Reply by mayabechlerspeicher** [reply]
Thank you.
Do you know where the mapping is available? I am having trouble finding it.

**Reply by rusty1s** [reply]
I don't really know, sorry.
